docs/development.rst

```diff
@@ -1,8 +1,18 @@
 Development
 ===========
 
-yamllint provides both a script and a Python module. The latter can be used to
-write your own linting tools:
+``yamllint`` provides both a script and a Python module. The latter can be used
+to write your own linting tools:
+
+Basic example of running the linter from python:
+
+.. code-block:: python
+
+   import yamllint
+
+   yaml_config = yamllint.config.YamlLintConfig("extends: default")
+   for p in yamllint.linter.run(".travis.yml", yaml_config):
+         print(p.desc, p.line, p.rule)
```

Comment:
6-space indentation is quite uncommon, can you please use 4 (the default for most Python projects)?

Reply:
LOL, vscode got confused by the rst 3 dots prefix. Fix.